website/views/bounty.py

```
@@ -0,0 +1,417 @@
+class PaymentData:
+    """Data structure to hold payment-related information."""
+
+    contributor_username: str
+    bounty_amount: int
+    pr_number: int
+    issue_number: int
+    owner_name: str
+    repo_name: str
+
+
+class PaymentData:
```

Pylint (3.3.7)

[error] class already defined line 17

(E0102)